shared/src/main/scala/mlscript/JSBackend.scala

```diff
@@ -152,29 +153,36 @@ class JSBackend(pgrm: Pgrm) {
       val letScope = Scope(name :: Nil, scope)
       JSImmEvalFn(
         name :: Nil,
-        Left(translateTerm(body)(letScope)),
+        letScope withTempVarDecls translateTerm(body)(letScope),
         translateTerm(value)(letScope) :: Nil
       )
     case Blk(stmts) =>
+      val blkScope = Scope(Nil, scope)
       JSImmEvalFn(
         Nil,
-        Right(stmts flatMap (_.desugared._2) map { translateStatement(_) }),
-        JSPlaceholderExpr() :: Nil
-      )
-    case CaseOf(term, cases) => {
-      val arg = translateTerm(term)
-      val param = scope.allocate()
-      JSImmEvalFn(
-        param :: Nil,
-        Right(translateCaseBranch(param, cases)(Scope(param :: Nil, scope))),
-        arg :: Nil
+        R(blkScope withTempVarDecls (stmts flatMap (_.desugared._2) map {
+          translateStatement(_)(blkScope)
+        })),
+        Nil
       )
-    }
-    case IntLit(value) => {
-      val useBigInt = MinimalSafeInteger <= value && value <= MaximalSafeInteger
-      JSLit(if (useBigInt) { value.toString }
-      else { value.toString + "n" })
-    }
+    // Pattern match with only one branch -> comma expression
+    case CaseOf(trm, Wildcard(default)) =>
+      println("what")
```

Comment:
Please remove all these lines of debug-printing.

Reply:
They were removed.